core/src/test/java/org/elasticsearch/env/NodeEnvironmentTests.java

```diff
@@ -300,7 +300,7 @@ public void run() {
     @Test
     public void testCustomDataPaths() throws Exception {
         String[] dataPaths = tmpPaths();
-        NodeEnvironment env = newNodeEnvironment(dataPaths, Settings.EMPTY);
+        NodeEnvironment env = newNodeEnvironment(dataPaths, "/tmp", Settings.EMPTY);
```

Comment:
just checking that this test does not create files in this dir, otherwise we should rather use the java.io.tmpdir?